README.md

```diff
@@ -379,6 +379,24 @@ du -hs e2e_tests/database/${SNAPSHOT_NAME}*
 
 Commit the result.
 
+### Deploying the reverse proxy initially
+
+```sh
+scp -C -r docker/reverseproxy/conf.d/ docker/reverseproxy/docker-compose.yml docker/reverseproxy/.env.server lzm:/tmp/reverseproxy
```

Comment:
What is stored in lzm:/tmp/reverseproxy? Where does that come from?

Reply:
Sadly we have multiple ways of referring to the server as an SSH destination in the README:
`monitoring@monitoring.localzero.net` and `lzm` which is an alias for the former.

So the line above just means "copy the stuff to /tmp/reverseproxy on the server".
I'll update it to contain the full name instead though, like in the other parts of the deployment steps.